package.json

```diff
@@ -1,16 +1,70 @@
-{
-  "name": "task1",
-  "version": "0.1.0",
-  "private": true,
-  "dependencies": {
-    "react": "^16.3.1",
-    "react-dom": "^16.3.1",
-    "react-scripts": "1.1.4"
-  },
-  "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test --env=jsdom",
-    "eject": "react-scripts eject"
-  }
-}
\ No newline at end of file
+{
+    "name": "task4",
+    "version": "1.0.0",
+    "description": "App built as task 2 for the react mentoring program",
+    "main": "index.js",
+    "scripts": {
+        "build": "webpack --config ./webpack.config.js --mode production",
+        "wds": "webpack-dev-server",
+        "start:dev": "npm run wds -- --config ./webpack.config.dev.js --mode development",
+        "start": "simplehttpserver dist",
+        "test": "jest",
+        "test:coverage": "npm run test -- --coverage",
+        "cypress:open": "cypress open"

```

Comment:
Is a good idea to have a script to just run `npm run test:e2e` and all required steps being executed.